README.md

```diff
@@ -1 +1,63 @@
-# toDo-Review
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To-Do List
+
+> A minimalist to-do list app taking advantage of webpack 
+
```

Comment:
- [ ] Optional Kindly use a description of your project, how it looks, what feature it has? For users to be interested in your app.

Reply:
Thank you for the review 😀. I will try and implement the optional suggestions in the README 👍